src/action.ts

```diff
@@ -98,6 +99,14 @@ async function action(pr: PullRequest): Promise<void> {
     ? { name: tmtPlanRegexParsed.data }
     : {};
 
+  // Conditionally include the name attribute only if tmt_plan_Filter is not null
+  const tmtPlanFilterParsed = tmtPlanFilterSchema.safeParse(
+    getInput('tmt_plan_filter')
+  );
+  const tmtPlanFilter = tmtPlanFilterParsed.success
+    ? { name: tmtPlanFilterParsed.data }
```

Comment:
Based on TF documentation, `plan_filter` accepts a string, not an object.

https://docs.testing-farm.io/Testing%20Farm/0.1/test-request.html#plan-filter

```suggestion
    ? tmtPlanFilterParsed.data
```